string camera_id


float32 confidence
